miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/context/MisoAppListener.java

```diff
@@ -173,39 +169,46 @@ private NamingScheme getConfiguredNamingScheme(String namespace, Map<String, Str
       return null;
     }
     NamingScheme scheme = resolver.getNamingScheme(schemeName);
-    if (scheme == null) throw new IllegalArgumentException("Failed to load naming scheme '" + schemeName + "'");
+    if (scheme == null)
+      throw new IllegalArgumentException("Failed to load naming scheme '" + schemeName + "'");
     SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(scheme);
 
-    applyGenerator(namespace + ".generator.nameable.name", misoProperties, resolver::getNameGenerator, scheme::setNameGenerator);
+    applyGenerator(namespace + ".generator.nameable.name", misoProperties, resolver::getNameGenerator,
+        scheme::setNameGenerator);
     applyGenerator(namespace + ".generator.sample.alias", misoProperties, resolver::getSampleAliasGenerator,
         scheme::setSampleAliasGenerator);
     applyGenerator(namespace + ".generator.library.alias", misoProperties, resolver::getLibraryAliasGenerator,
         scheme::setLibraryAliasGenerator);
-    applyGenerator(namespace + ".generator.libraryaliquot.alias", misoProperties, resolver::getLibraryAliquotAliasGenerator,
+    applyGenerator(namespace + ".generator.libraryaliquot.alias", misoProperties,
+        resolver::getLibraryAliquotAliasGenerator,
         scheme::setLibraryAliquotAliasGenerator);
 
-    applyValidator(namespace + ".validator.nameable.name", misoProperties, resolver::getNameValidator, scheme::setNameValidator);
+    applyValidator(namespace + ".validator.nameable.name", misoProperties, resolver::getNameValidator,
+        scheme::setNameValidator);
     applyValidator(namespace + ".validator.sample.alias", misoProperties, resolver::getSampleAliasValidator,
         scheme::setSampleAliasValidator);
     applyValidator(namespace + ".validator.library.alias", misoProperties, resolver::getLibraryAliasValidator,
         scheme::setLibraryAliasValidator);
-    applyValidator(namespace + ".validator.libraryaliquot.alias", misoProperties, resolver::getLibraryAliquotAliasValidator,
+    applyValidator(namespace + ".validator.libraryaliquot.alias", misoProperties,
+        resolver::getLibraryAliquotAliasValidator,
         scheme::setLibraryAliquotAliasValidator);
     applyValidator(namespace + ".validator.project.shortName", misoProperties, resolver::getProjectShortNameValidator,
```

Comment:
Should probably change `".validator.project.shortName"` to `".validator.project.code"` for consistency, and I believe `resolver::getProjectShortNamevalidator` should be `resolver::getProjectCodeValidator`